packages/in_app_purchase/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 0.6.0
+
+As part of implementing federated architecture and making the interface compatible for other platform this version contains the following **breaking changes**:
+
+* Changes to the platform agnostic interface:
+  * The `InAppPurchaseConnection` class has been renamed to `InAppPurchase`. To access the generic InApp Purchase API use `InAppPurchase.instance`;
+  * The `purchaseUpdatedStream` has been renamed to `purchaseStream`. Restored purchases are now also emitted on this stream and are marked with a `status` of `PurchaseStatus.restored`;
```

Comment:
We don't have to mention "restored purchases are now also emitted on this stream and are marked with a `status` of `PurchaseStatus.restored`;"
Instead, we should give more details in the item below: 

The `InAppPurchaseConnection.queryPurchases` method has been removed. Instead, you should use `InAppPurchase.restorePurchases`. This method emits each restored purchase on the `InAppPurchase.purchaseStream`, the purchase object will be marked with a `status` of `PurchaseStatus.restored`

Also, for each API, mention its class, for example, `InAppPurchaseConnection.purchaseUpdatedStream`, and `InAppPurchase.purchaseStream`

Ditto below for all the other items.

Reply:
done